core/dbt/cli/flags.py

```diff
@@ -76,6 +77,11 @@ def args_to_context(args: List[str]) -> Context:
     return sub_command_ctx
 
 
+DEPRECATED_PARAMS = {
+    "deprecated_print": "print",
+}
```

Comment:
Will the approach here be extensible to other deprecated/renamed flags & env vars? (#6903)

Reply:
Yes.